wolfssl/internal.h

```diff
@@ -4465,6 +4465,7 @@ WOLFSSL_LOCAL int SetKeysSide(WOLFSSL*, enum encrypt_side);
 
 #ifdef HAVE_ECC
     WOLFSSL_LOCAL int EccMakeKey(WOLFSSL* ssl, ecc_key* key, ecc_key* peer);
+    WOLFSSL_LOCAL unsigned char GetCurveByOID(int oidSum);
```

Comment:
Maybe the return code should be word16?

Reply:
I can update it to that, looking at the value of WOLFSSL_FFDHE_8192 I think using word16 instead would be a good idea. The choice of "unsigned char" was only to keep it the same as the original SetCurveId function.

Reply:
Or better yet maybe just "unsigned int" here.